### PR TITLE
Allow .js setting files

### PR DESCRIPTION
**Pull request by afcapel**
Connects to https://github.com/sharelatex/analytics-sharelatex/issues/189

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "settings-sharelatex",
   "homepage": "www.sharelatex.com",
   "description": "A centralised settings system for ShareLaTeX",
-  "version": "1.0.0",
+  "version": "1.0.1",
```
Comment (ShaneKilkelly) [inline package.json]: I never thought I'd be the guy who quibbles over semver numbers, but this change feels like a `1.1.0` :smile:

Comment (afcapel) [reply package.json]: Good catch, fixed in 75c812eecd6b6803de38a2f68f4d07edc63da064

**Comment by ShaneKilkelly** [comment]
Connects to https://github.com/sharelatex/analytics-sharelatex/issues/189

**Comment by afcapel** [comment]
This has been working on analytics for a while, but it needs merging into master and deploy to other services.

**Comment by natestemen** [comment]
Now that we are starting to decaffeinate more repos, it'll be way easier to merge this into master and update the `settings-sharelatex` version as we go.

Just as reference I ran into an issue with trying to run acceptance test on decaffed `chat-sharelatex`.

**Pull request merged**